TLM/TLM/UI/RoadSelectionPanels.cs

```diff
@@ -355,8 +359,21 @@ public abstract class ButtonExt : BaseMenuButton {
                 public void Refresh() {
                     isEnabled = !ShouldDisable();
                     UpdateButtonImageAndTooltip();
+                    width = height = GetButtonSize();
+                    Show();
                 }
 
+                /// <summary>Define size as smallest of 2.08% of width or 3.7% of height (40 px at 1080p).
+                /// The button cannot be smaller than 40px.</summary>
+                /// <returns>Button size for current screen resolution.</returns>
+                internal static float GetButtonSize() {
+                    var scaledSize
+                        = U.UIScaler.ScreenSizeSmallestFraction(0.0208f, 0.037f) *
+                          U.UIScaler.GetUIScale();
```

Comment:
Suggested change (i will be applying similar change in my code too) makes it easier to see the origin of the fraction constants:
```
const float SUGGESTED_SIZE = 40f;
const float FRAC_X = SUGGESTED_SIZE / 1920f;
const float FRAC_Y = SUGGESTED_SIZE / 1080f;
...
U.UIScaler.ScreenSizeSmallestFraction(FRAC_X, FRAC_Y)
```